Assets/_Functionalities/AssetBundles/Scripts/AssetBundleLoader.cs

```diff
@@ -110,14 +111,36 @@ private IEnumerator GetAssetBundle(string path, UnityAction<AssetBundle> callBac
             if (request.result != UnityWebRequest.Result.Success)
             {
                 Debug.LogError(request.error);
-                callBack?.Invoke(null);
             }
             else
             {
                 // Get downloaded asset bundle
                 AssetBundle bundle = DownloadHandlerAssetBundle.GetContent(request);
                 callBack?.Invoke(bundle);
             }
+#else
+            // Use AssetBundle.LoadFromFile for the Editor
+            if (File.Exists(path))
```

Comment:
Waarvoor is dit nodig? Ik weet niet beter of UnityWebRequest kan ook prima overweg met lokale assetbundles en paden. Is er een specifieke error die dit oplost die ik niet overzie?

Reply:
dat werkt niet in de editor, want ik kreeg errors